### PR TITLE
chore: catch new crates and for syn 2

### PR DESCRIPTION
**Pull request by tisonkun**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "logcall"
-version = "0.1.10"
+version = "0.1.11"
```
Comment (tisonkun) [inline Cargo.toml:3]: Not sure if bump to 0.2.

Comment (andylokandy) [reply Cargo.toml:3]: https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api

Semver suggest not to bump major when upgrading dependencies

**Comment by andylokandy** [comment]
Thank you!

**Comment by andylokandy** [comment]
Published 0.1.11

**Pull request merged**